eureka-core/src/main/java/com/netflix/eureka/EurekaServerConfig.java

```diff
@@ -408,6 +408,13 @@
      * @return maximum number of threads to be used for replication.
      */
     int getMaxThreadsForPeerReplication();
+    
+    /**
+     * Get the minimum number of available peer replication instances
+     * 
+     * @return minimum number of available peer replication instances
+     */
+    int getMinNumberOfAvailablePeers();
```

Comment:
Looks like this config is designed for the statusPage only. Perhaps rename the method to reflect that? As it stands, it (name + comment) feels more like something that is system critical.